Extensions/BitBucket/Src/vss-extension.json

```diff
@@ -1,9 +1,9 @@
 {
-  "manifestVersion": 1.0,
+  "manifestVersion": 2.0,
   "id": "vss-services-bitbucket",
   "name": "Bitbucket artifacts for Release Management",
   "publisher": "ms-vscs-rm",
-  "version": "15.157.0",
+  "version": "15.157.1",
```

Comment:
Not sure that we have to release "new version" of an extension. Probably just manifestVersion will be enough.

Reply:
According to Scott:

```
the manifestVersion value is not the version of the extension - it defines the format of the file itself.  so we don't want to change that.  what we want is to change the version , which is the version of the extension and what the gallery/EMS will look at to determine whether it changed.  in your PR you have it bumped to 15.157.1, which should be sufficient.
so i think you should undo the change to manifestVersion and then we can merge your PR.  after that we'll have to update it in the gallery
```

So I've rolled back manifestVersion change.